agent/installer/installer.go

```diff
@@ -85,6 +91,36 @@ func (bd *bundleDownloader) DownloadOrPreview(os, k8s, tag string) error {
 	return bd.Download(os, k8s, tag)
 }
 
+func ckeckPreRequsitePackages() error {
```

Comment:
Being a bit pedantic, can we move these 2 functions to something like `checks.go`? Don't want to bloat the `installer.go` file